### DISCUSSION TITLE
Waiting for check wich is not in the config anymore

### DISCUSSION THREAD

**Discussion by smutel** [community/community#26763] [Actions]
<p>Hello,</p>
<p>I created this PR on my own project : <a href="https://github.com/smutel/terraform-provider-netbox/pull/85" class="inline-onebox" rel="noopener nofollow ugc">fix: Error message when vcpus is not specified by smutel · Pull Request #85 · smutel/terraform-provider-netbox · GitHub</a>.</p>
<p>I don’t understand why a check called</p>
<pre><code class="lang-auto">LintAndBuild (1.15.x) Expected — Waiting for status to be reported
</code></pre>
<p>is still there …</p>
<p>In my config <a href="https://github.com/smutel/terraform-provider-netbox/blob/df09681cb6e99fcf9607d5c1e06bd1569dec2d24/.github/workflows/pr.yml#L13" class="inline-onebox" rel="noopener nofollow ugc">terraform-provider-netbox/pr.yml at df09681cb6e99fcf9607d5c1e06bd1569dec2d24 · smutel/terraform-provider-netbox · GitHub</a>, we can see that I want only golang 1.16.x.</p>
<p>Thanks.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>It looks like that check is marked as required in branch protection rules. I assume the required check wasn’t updated with the golang update.</p>
